src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -435,15 +447,15 @@ public String getName()
 	}
 
 	@Override
-	public String getVersion()
+	public @NotNull String getVersion()
 	{
-		return String.format("MockBukkit (MC: %s)", BUKKIT_VERSION);
+		return String.format("MockBukkit (MC: %s)", getBukkitVersion());
 	}
 
 	@Override
-	public String getBukkitVersion()
+	public @NotNull String getBukkitVersion()
 	{
-		return BUKKIT_VERSION;
+		return buildProperties.getProperty("full-api-version").split("-")[0];
```

Comment:
If for whatever reason the file failed to read, `buildProperties.getProperty("full-api-version")` will be nullable and calling `String#split` is gonna throw a NPE.
I think it would be better to add a null check here and throw a more meaningful exception, e.g. `IllegalStateException("Bukkit version could not be determined")`

Reply:
Done